### ISSUE TITLE
Adapt disabled tests checking debug.getinfo() activelines from PUC-Rio Lua 5.1 test suite in db.lua

### ISSUE THREAD

**Issue by Buristan** [tarantool/tarantool#5708]
This ticket is a logical continuation of gh-4473.

If run  PUC-Rio Lua 5.1 test suite by LuaJIT (HEAD at the moment: [`1672bdc`](https://github.com/tarantool/luajit/commit/1672bdc0ffbee9f32fadb6943909d6af7eb9a7b1)) (not Tarantool) it fails with the following error:
```
/home/burii/builds_workspace/luajit/gh-4473-adopt-luavela-test-suites/src/luajit: db.lua:463: assertion failed!
stack traceback:
        [C]: in function 'assert'
        db.lua:463: in function 'dofile'
        ...adopt-luavela-test-suites/test/PUC-Lua-5.1-tests/all.lua:82: in main chunk
        [C]: at 0x55fc9011ff96
```
caused by this test chunk (assuming that all previous failing test chunks non relating to this error are disabled):
```lua
co = coroutine.create(function (x)
       local a = 1
       coroutine.yield(debug.getinfo(1, "l"))
       coroutine.yield(debug.getinfo(1, "l").currentline)
       return a
     end)

local tr = {}
local foo = function (e, l) table.insert(tr, l) end
debug.sethook(co, foo, "l")

local _, l = coroutine.resume(co, 10)
local x = debug.getinfo(co, 1, "lfLS")
assert(x.currentline == l.currentline and x.activelines[x.currentline])
assert(type(x.func) == "function")
for i=x.linedefined + 1, x.lastlinedefined do
  assert(x.activelines[i]) -- this assertion failed
  x.activelines[i] = nil
end

```

LuaJIT does not report line with single "end" statement as an active line in `debug.getinfo()`, unlike Lua does.

**Comment by Buristan** [comment]
This test chunk can be adapted to LuaJIT behavior like this:
```diff
diff --git a/test/PUC-Lua-5.1-tests/db.lua b/test/PUC-Lua-5.1-tests/db.lua
index 632efcc..b467fb1 100644
--- a/test/PUC-Lua-5.1-tests/db.lua
+++ b/test/PUC-Lua-5.1-tests/db.lua
@@ -463,9 +463,9 @@ assert(type(x.func) == "function")
 -- LuaJIT does not report line with single "end" statement as
 -- an active line in debug.getinfo(), unlike Lua does.
 -- See also https://github.com/tarantool/tarantool/issues/5708.
--- Test is disabled for LuaJIT for now.
-for i=x.linedefined + 1, x.lastlinedefined do
-  -- assert(x.activelines[i])
+-- Test is adapted to the behaviour of LuaJIT.
+for i=x.linedefined + 1, x.lastlinedefined - 1 do
+  assert(x.activelines[i])
   x.activelines[i] = nil
 end
 assert(next(x.activelines) == nil)   -- no 'extra' elements
 ```

**Comment by igormunkin** [comment]
The issue is resolved in scope of tarantool/luajit@67c41b7. The submodule was updated in tarantool in 2.8.0-249-g8a09e18ce (8a09e18), 2.7.1-210-g5f4298b6f (5f4298b), 2.6.2-203-g8343a8129 (8343a81), 1.10.9-119-g7a4c7d0b0 (7a4c7d0).
